src/app.ts

```diff
@@ -487,11 +532,22 @@ function registerAPIRoutes(
   );
 }
 
-function registerAuthenticationRoutes(app: Express, basePath: string): void {
+function registerAuthenticationRoutes(
+  app: Express,
+  basePath: string,
+  sessionStorage: RedisSessionStorage,
+  samlCert: string,
+  tokenService: TokenService,
+  getRedirectionUrl: (token: string) => UrlFromString
+): void {
   const bearerTokenAuth = passport.authenticate("bearer", { session: false });
 
-  const acsController: AuthenticationController = container.resolve(
-    AUTHORIZATION_CONTROLLER
+  const acsController: AuthenticationController = new AuthenticationController(
+    sessionStorage,
+    samlCert,
+    defaultModule.SPID_STRATEGY,
```

Comment:
this should be passed as parameter, not retrieved from a global var

Reply:
fixed in 5650e7f